shared/django_apps/codecov_auth/models.py

```diff
@@ -381,7 +382,9 @@ class Meta:
     )
 
     objects = OwnerManager()
-
+    tracker = FieldTracker(
+        fields=["username", "service", "upload_token_required_for_public_repos"]
```

Comment:
This is fine, although if you don't specify the fields, then you can track any field you want, so you wouldn't have to make a future PR for this. Pros and cons, this makes it more specific too. I'll leave this up to you!

Reply:
yeah I hear that - at the moment I'd like to keep it as slim as possible, so I'll keep it with these specific fields.